src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/form.html.twig

```diff
@@ -85,7 +85,7 @@
       }) }}
 
       {% set keywordHint %}
-        {{ 'To add "tags", click inside the field, write something, and then press "Enter".'|trans({}, 'Admin.Catalog.Help') }}
+        {{ 'To add tags, click in the field, write something, and then press the "Enter" key.'|trans({}, 'Admin.Catalog.Help') }}
```

Comment:
In order to avoid duplicate, please localize it in _Admin.Shopparameters.Help_.

Reply:
Done